Dockerfile.build-tools

```diff
@@ -111,6 +111,34 @@ RUN for package in Capture::Tiny DateTime Devel::Cover Digest::MD5 File::Spec JS
     && cd lcov \
     && make install \
     && rm -rf ../lcov.tar.gz
+# Compile and install the static OpenSSL library
+ENV OPENSSL_VERSION=3.2.0
+ENV OPENSSL_PREFIX=/usr/local/openssl
+RUN wget -O /tmp/openssl-${OPENSSL_VERSION}.tar.gz https://www.openssl.org/source/openssl-${OPENSSL_VERSION}.tar.gz && \
+    echo "14c826f07c7e433706fb5c69fa9e25dab95684844b4c962a2cf1bf183eb4690e  /tmp/openssl-${OPENSSL_VERSION}.tar.gz" | sha256sum --check && \
```

Comment:
Do our SBOM scanning tools detect this?

Reply:
its not detecting somehow or not enabled for this repo? also I'm not aware about this tool. 

@areyou1or0 Can you please confirm this?

Reply:
I configured Arnica to prompt issues for severity High and Critical to reduce noise. That's why you don't see any pop-up here.

Reply:
We need some mechanism to ensure the version is not out of date. If Arnica can be used for it it's good, if it can't be configured well enough to get that signal out of the noise, then we'll need something else.

Autodetecting and then just taking the latest version isn't enough, because the dockerfile is not rebuilt each time from my understanding, but only if it is changed in git.

Reply:
I'll see if I can customize the alerts on Arnica, thanks for flagging this.